### DISCUSSION TITLE
Regarding the Dice problem in the log printed after each Epoch

### DISCUSSION THREAD

**Discussion by GrantMiraitowa** [MIC-DKFZ/nnUNet#888] [Q&A] [answered]
Hello, is the Dice obtained after each Epoch in the log printed by nnUNet a training Dice or a verification Dice? This is very important to me, thank you!

**Comment by FabianIsensee** [comment, marked as answer]
Please just read the text that is displayed after each epoch...
It is also always worth looking into the documentation, especially when something is important...

> Also shows an approximation of the evlauation metric (green). This approximation is the average Dice score of the foreground classes. It should, however, only to be taken with a grain of salt because it is computed on randomly drawn patches from the validation data at the end of each epoch, and the aggregation of TP, FP and FN for the Dice computation treats the patches as if they all originate from the same volume ('global Dice'; we do not compute a Dice for each validation case and then average over all cases but pretend that there is only one validation case from which we sample patches). The reason for this is that the 'global Dice' is easy to compute during training and is still quite useful to evaluate whether a model is training at all or not. A proper validation is run at the end of the training.
